apinotes/Foundation.apinotes

```diff
@@ -659,6 +659,9 @@ Classes:
   - Selector: 'URLForPublishingUbiquitousItemAtURL:expirationDate:error:'
     SwiftName: url(forPublishingUbiquitousItemAt:expiration:)
     MethodKind: Instance
+  - Selector: 'enumeratorAtURL:includingPropertiesForKeys:options:errorHandler:'
+    MethodKind: Instance
+    Availability: nonswift
```

Comment:
Instead of `Availability: nonswift`, please use `SwiftPrivate: true`. That way it doesn't show up in overload resolution.